src/btree/bt_sync_obsolete.c

```diff
@@ -501,6 +529,15 @@ __checkpoint_cleanup_eligibility(WT_SESSION_IMPL *session, const char *uri, cons
       (newest_stop_durable_ts != WT_TS_NONE || logged || strcmp(uri, WT_HS_URI) == 0))
         return (true);
 
+    /*
```

Comment:
Should we separate the previous checks into multiple checks and comments? Otherwise, the comment here seems odd as it's not in the situations listed before.

Reply:
Done.

Reply:
This is really good like this!